Portal/src/Datahub.Core/Configuration/DataProjectsConfiguration.cs

```diff
@@ -22,6 +22,7 @@ public record DataProjectsConfiguration
     public bool IoT { get; set; } = true;
     public bool StorageCost { get; set; } = true;
     public bool ComputeCost { get; set; } = true;
+    public bool VirtualMachines { get; set; } = false;
```

Comment:
Is this going to be superseded by whitelisting feature?

Reply:
Yeah exactly, I just wanted the card out